spec/factories/tasks.rb

```diff
@@ -26,7 +27,7 @@
   factory :task, class: 'Task' do
     title { Faker::Lorem.sentence }
     description { Faker::Lorem.paragraph }
-
+    due_date { rand(1.month.ago..1.month.since)}
```

Comment:
also I don' want to have random due_dates
put nil

and add 2 traits with "expired" and "future" due dates